collect_app/src/main/res/values/untranslated.xml

```diff
@@ -16,4 +16,9 @@ the specific language governing permissions and limitations under the License. -
     <string name="default_odk_formlist" translatable="false">/formList</string>
     <string name="default_odk_submission" translatable="false">/submission</string>
     <string name="app_url" translatable="false">http://opendatakit.org</string>
+    <string name="never_value" translatable="false">never</string>
```

Comment:
They are in this file because it allows me to use them in arrays https://github.com/opendatakit/collect/pull/2095/files#diff-7bc0b360d7b3721e7e51eed52ca8ea64R177 but in other places too.